fontforge/parsettf.c

```diff
@@ -3999,7 +3999,7 @@ return( 0 );
 	    SCConvertToOrder2(info->chars[i]);
     }
 
-    for ( i=0; fontnames[i]!=NULL && i<1; ++i ) {
+    for ( i=0; i<1 && fontnames[i]!=NULL ; ++i ) {
```

Comment:
Hmm. It's from the initial revision of FontForge, so it's unclear whether this was a fix of some sort.